src/Exceptionless.Core/Repositories/StackRepository.cs

```diff
@@ -26,6 +26,14 @@ public StackRepository(ExceptionlessElasticConfiguration configuration, IValidat
             return FindAsync(q => q.ElasticFilter(Query<Stack>.DateRange(d => d.Field(f => f.SnoozeUntilUtc).LessThanOrEquals(utcNow))), options);
         }
 
+        public Task<FindResults<Stack>> GetStacksForCleanupAsync(DateTime utcNow, CommandOptionsDescriptor<Stack> options = null) {
+            return FindAsync(q => q
+                .ElasticFilter(Query<Stack>.DateRange(d => d.Field(f => f.LastOccurrence).LessThanOrEquals(utcNow)))
+                .FieldEquals(f => f.Status, StackStatus.Open)
+                .FieldEmpty(f => f.References)
```

Comment:
Is this the only thing we need to check for custom data put on the stack?

Reply:
The only custom data we have on a stack is reference links